manifests/third-party/jobset/kustomization.yaml

```diff
@@ -12,3 +12,10 @@ patches:
       group: ""
       version: v1
       kind: Namespace
+  # Add security context to jobset-controller-manager.
```

Comment:
Can we propose these changes to the JobSet upstream if @kannon92 and @ahg-g doesn't have any objections?
I can see that LWS also enforce PSS: https://github.com/kubernetes-sigs/lws/blob/main/config/manager/manager.yaml#L60

Reply:
We already have these.

Reply:
https://github.com/kubernetes-sigs/jobset/blob/main/config/components/manager/manager.yaml#L75

And the helm chart always you to inject your own PSS.

https://github.com/kubernetes-sigs/jobset/blob/main/charts/jobset/values.yaml#L59

Reply:
Should we change the default helm chart security context values then to be PSS restricted compatible?

Reply:
Who is that question towards @juliusvonkohout?

Reply:
The maintainers. We should fix it here for Helm and Kustomize, since we only support Kubernetes 1.31+ https://github.com/kubernetes-sigs/jobset/blob/fd0a275542e2abd56584963659dde63d16609c51/config/components/manager/manager.yaml#L67 and https://kubernetes.io/releases/ for supported releases. So lets add the seccomprofile: default

Reply:
To @kunal-511 can you modify also fix helm charts values accordingly ?